docs/tutorial/GetStarted.md

```diff
@@ -175,9 +179,9 @@ docker-user-name/time-server      part1       42somsoc147
 #### Test your container
 When you execute `docker images`, you should be able to see the your image in the displayed list. To make sure the image is working as expected, you can run the container by executing the below command :
 
-`docker run -env PORT0=8000 -p 80:8000 -t docker-user-name/time-server:part1`
+`docker run --env PORT0=8000 -p 80:8000 -t docker-user-name/time-server:part1`
 
-Note that you are reading the port number from the `PORT0` environment variable. Marathon sets this environment variable when launching the container and since you don't have that yet, you need to provide the environment variable manually using `-env PORT0=8000`. The `-p` option maps the host port 80 to the container port 8000. The  `-t` flag creates a pseudoTTY and since you unbuffered the Python standard I/O in your Dockerfile, you will be able to see the real time logs of the server in the console. Once you executed the above command, you should be able to browse [localhost](http://localhost:80).
+Note that you are reading the port number from the `PORT0` environment variable. Marathon sets this environment variable when launching the container and since you don't have that yet, you need to provide the environment variable manually using `--env PORT0=8000`. The `-p` option maps the host port 80 to the container port 8000. The  `-t` flag creates a pseudoTTY and since you unbuffered the Python standard I/O in your Dockerfile, you will be able to see the real time logs of the server in the console. Once you have executed the above command, you should be able to browse [localhost](http://localhost:80). You can test the url with `curl localhost:8000` and your server should return the current time.
```

Comment:
suggest:

**Note:** You must set the value of `PORT0` explicitly because Marathon sets this value when it launches the container, which has not happened yet.